### DISCUSSION TITLE
Keep custom filters after DBeaver version update

### DISCUSSION THREAD

**Discussion by bombillazo** [dbeaver/dbeaver#35759] [Q&A]
Hey, I recently upgraded DBeaver versions and noticed all my custom filters for the tables were gone. How can I ensure a version update does not clear that type of settings?

**Comment by TB-HunBoka** [comment]
It should be stored in this file:
AppData\Roaming\DBeaverData\workspace7\.metadata\.plugins\org.jkiss.dbeaver.core\

Sometimes I make a copy of these folders to make sure I have a backup, but I saw that the workspace number has been increased from 6 to 7, so you should have a workspace6 folder too. Try to check the file there.

**Reply by bombillazo** [reply]
Im in MacOS

**Reply by TB-HunBoka** [reply]
Workspace locations for different OS.
https://dbeaver.com/docs/dbeaver/Workspace-Location/
MacOS 	~/Library/DBeaverData/
